### ISSUE TITLE
Simplification and/or documentation of `tools/development`

### ISSUE THREAD

**Issue by codepunkt** [ctrlplusb/react-universally#311]
So far, i think i understand most of this repositories inner workings.

The only part where it gets tricky for me is in `tools/development` regarding hot reloading of client modules, reloading of the server, the order in which that has to happen, all the dispose thing with closing open connections etc.

That seems rather complicated to me - or at least not as well documented (in code and in readmes) as the rest of the code base. I'd love to find a way to simplify this or document it better.

**Comment by ctrlplusb** [comment]
Hey @codepunkt

Yeah, agreed, that part of the project is not up to the same standards as the rest.  My thoughts were that this is more to do with the internals of the system and I didn't anticipate that anyone would need to modify it.

Have you been finding requirements to dig into the development tools and make adjustments? If so, what sort of adjustments have you had to make?

I am happy to do some work on this.  I'll accept this.

**Comment by codepunkt** [comment]
You are right - i did not have the need to modify this part because so far it just works. I simply like to understand the codebase i'm basing stuff on and therefore love the documentation-heavy approach this repository takes ;-)

**Comment by ctrlplusb** [comment]
👌
